graphql/util.go

```diff
@@ -1290,3 +1290,63 @@ func getAPISubscriptionsForProject(ctx context.Context, projectId string) ([]*re
 	}
 	return res, nil
 }
+
+func GroupProjects(projects []model.ProjectRef, onlyDefaultedToRepo bool) ([]*GroupedProjects, error) {
```

Comment:
Can we add a function comment here to explain what's happening? (And can this be a private function?)

Reply:
done

Reply:
it's there now